alerts.c

```diff
@@ -68,6 +69,21 @@ alerts_callback(__unused int fd, __unused short events, __unused void *arg)
 }
 
 static int
+alerts_action_applies(struct winlink *wl, const char *name)
+{
+	int	action;
+
+	action = options_get_number(wl->session->options, name);
+	if (action == ALERT_ANY)
+		return (1);
+	if (action == ALERT_CURRENT)
+		return (wl == wl->session->curw ? 1 : 0);
```

Comment:
Acknowledged. I saw this usage elsewhere (like window.c), and I was trying hard to adhere to the existing style. I'll modify it.